examples/oneapi-samples/src/1d_HeatTransfer/src/1d_HeatTransfer.cpp

```diff
@@ -0,0 +1,294 @@
+﻿//==============================================================
```

Comment:
Could we avoid copying of the src via a git clone preparation step?

Reply:
Possibly later, but ATM it does not make sense. The oneapi-samples git repo has an unusable (for PoCL) buildsystem (Cmake files with hardcoded `icpx` compiler, sometimes not even CMake just plain handwritten Makefile, each sample directory is a separate CMake / Make project ... etc), so we have to use our own CMake files anyway. If the oneapi-samples  buildsystem is fixed someday, then we could use the repo directly.

Reply:
Can you document the reasons for copying in the commit log?